src/BlockTypesController.php

```diff
@@ -178,6 +178,7 @@ protected function get_block_types() {
 			'ProductTitle',
 			'ProductSummary',
 			'ProductStockIndicator',
+			'ProductSaleBadge',
```

Comment:
I think a comment here explaining why we register an atomic block here will be helpful.

Reply:
Good point! I added a `@todo comment`. In this way, in the end, I will re-organize the order of the elements in the array (maybe we can move down the atomic blocks and add a comment that explains why)